packages/ember-metal/lib/injected_property.js

```diff
@@ -24,8 +24,20 @@ export default class InjectedProperty extends ComputedProperty {
     super(injectedPropertyGet);
 
     this.type = type;
-    this.name = name;
     this.source = options ? options.source : undefined;
+
+    if (name) {
+      let namespaceDelimiterOffset = name.indexOf('::');
+      if (namespaceDelimiterOffset === -1) {
+        this.name = name;
+        this.namespace = undefined;
+      } else {
```

Comment:
Let’s tweak this else case  to ensure it’s feature flagged

Reply:
This is already guarded in: https://github.com/emberjs/ember.js/pull/16240/files#diff-11c397e816cb737423d40c0a91104f5fR40

`packages/ember-runtime/lib/inject.js` ensures that `options` is only passed if the feature flag is enabled 👍

Reply:
Awesome, thank you